components/Button.vue

```diff
@@ -0,0 +1,106 @@
+<template>
+   <link href='https://fonts.googleapis.com/css?family=Poppins' rel='stylesheet'>
+   <button :class ="[
+      primary ? (disabled ? 'primary-disabled' : 'primary-default') : (disabled ? 'secondary-disabled' : 'secondary-default'), 
+      'baseButton',
+      size === 'small' ? 'small' : '',
+      size === 'medium' ? 'medium' : '',
+      size === 'large' ? 'large' : '',
+      ]">{{ buttonText }}</button>
+</template>
+
+<script>
+export default {
+   props: {
+      buttonText: {
```

Comment:
Personally, I would prefer this to be `text` since `<Button buttonText="hello">` is a bit redundant.